guides/core_concepts/0010-defining-a-pipeline.md

```diff
@@ -112,3 +112,14 @@ astarte_devices_source
     .realms([{realm: ${$.config.realm}, jwt: ${$.config.pairing_jwt}}])
     .interfaces(${$.config.interfaces})
 ```
+
+# Installing a Pipeline
+
+Pipelines can be installed using Astarte Flow [REST API](api/index.html) or using Astarte Dashboard.
```

Comment:
use `api/index.html#/pipelines/createPipeline` as link so it takes the user directly to the relevant REST operation